pymc3/step_methods/hmc/nuts.py

```diff
@@ -90,7 +91,8 @@ class NUTS(BaseHMC):
     def __init__(self, vars=None, Emax=1000, target_accept=0.8,
                  gamma=0.05, k=0.75, t0=10, adapt_step_size=True,
                  max_treedepth=10, on_error='summary', **kwargs):
-        R"""
+        R"""Set up the No-U-Turn sampler.
```

Comment:
Actually, that is a funny thing to document in `__init__` since it is an attribute it provides on the `trace` object.  It makes sense to me to leave it where it is, and not adding an `Attribute` section.